pkgs/by-name/ex/extract_otp_secrets/package.nix

```diff
@@ -0,0 +1,93 @@
+{
+  lib,
+  copyDesktopItems,
+  fetchFromGitHub,
+  makeDesktopItem,
+  python3Packages,
+}:
+
+let
+  pyPkgs = python3Packages.overrideScope (
+    self: super: {
+      opencv4 = super.opencv4.override {
+        enableGtk3 = true;
```

Comment:
I suspect that this is not necessary (or even harmful) on darwin but cannot test it myself.